testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/slavereconnect/SlaveReconnectTestCase.java

```diff
@@ -198,9 +201,12 @@ private boolean checkHostServersStarted(DomainClient masterClient, String host)
                 String server = entry.asString();
                 op = Util.createEmptyOperation(READ_ATTRIBUTE_OPERATION, PathAddress.pathAddress(HOST, host).append(SERVER, server));
                 op.get(NAME).set("server-state");
-                DomainTestUtils.executeForResult(op, masterClient); // success is enaught
+                ModelNode state = DomainTestUtils.executeForResult(op, masterClient);
+                if (SUCCESS.equals(state.get(OUTCOME).asString())) {
+                    return "running".equals(state.get(RESULT).asString());
```

Comment:
I'd changed a bunch of these in the previous, and was getting a lot of failures with the wildfly mixed  domain tests, I'll look at these again, maybe they're safe.